lib/qrda-import/patient_importer.rb

```diff
@@ -39,6 +39,8 @@ def initialize
         @data_element_importers << generate_importer(MedicationDischargeImporter)
         @data_element_importers << generate_importer(MedicationDispensedImporter)
         @data_element_importers << generate_importer(PatientCharacteristicExpired)
+        @data_element_importers << generate_importer(PhysicalExamOrderImporter)
```

Comment:
same comment as above, suggest moving physicalexamperformedimporter to be with these